packages/cli/lib/ui.js

```diff
@@ -31,7 +31,9 @@ const uiLink = (linkText, url, options = {}) => {
     ].join('');
     return options.useColor ? chalk.cyan(result) : result;
   } else {
-    return options.fallback ? `${linkText}: ${url}` : linkText;
+    return options.useColor
```

Comment:
I think we should get rid of this `options.useColor` option. We should use color in every terminal that supports it.